src/test/java/org/minions/devfund/lourdes/MovieDatabaseTest.java

```diff
@@ -0,0 +1,136 @@
+package org.minions.devfund.lourdes;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Implemented tests for {@link MovieDatabase}.
+ */
+public class MovieDatabaseTest {
+    private static final String ACTOR_NAME = "Bruce Willis";
+    private static final String MOVIE_NAME_1 = "Sexto sentido";
+    private static final String MOVIE_NAME_2 = "Duro de Matar";
+    private static final String[] ACTORS_1 = new String[]{ACTOR_NAME, "Haley Joel"};
+    private static final String[] ACTORS_2 = new String[]{ACTOR_NAME, "Alam Rickman"};
+    private MovieDatabase movieDatabase;
+
+    /**
+     * Method that setup all initial values.
+     */
+    @Before
+    public void setup() {
+        movieDatabase = new MovieDatabase();
+    }
+
+    /**
+     * Verify add new movie to the list.
+     */
+    @Test
+    public void testAddMovie() {
+        movieDatabase.addMovie(MOVIE_NAME_1, ACTORS_1);
+        assertEquals(movieDatabase.getMovieList().get(0).getName(), MOVIE_NAME_1);
```

Comment:
Assertion arguments should be passed in the correct order
first argument is the expected result and the second one is the actual result.
Please update all your assertEquals